app/services/user/create.rb

```diff
@@ -0,0 +1,20 @@
+class User::Create < ApplicationService
+  def initialize(params)
+    @user = params[:user]
+  end
+
+  def call
+    create_profile(@user) if save_user(@user)
+    @user
+  end
+
+  protected
```

Comment:
What is a difference between `protected` and `private`? Why did you choose `protected` here?